packages/dds/cell/package.json

```diff
@@ -1,98 +1,102 @@
 {
```

Comment:
This is hard to review. Please roll back the formatting on all package.json files.

Reply:
git command to do this:

```
git checkout $(git merge-base HEAD main) **/package.json
```

note this will overwrite current changes you have to package.json files. it won't commit anything though, so you can always undo.

Reply:
(this reverts package.json files. if you need legitimate changes to more than a couple of them, you might want to use a formatting command instead--I think we're formatting them now?)